scripts/create-instance.sh

```diff
@@ -23,77 +23,77 @@
     echo "[ERROR] Example: export INVENIO_WEB_VENV=invenio3"
     exit 1
 fi
-if [ -v ${INVENIO_USER_EMAIL} ]; then
+if [[ -v INVENIO_USER_EMAIL ]]; then
     echo "[ERROR] Please set environment variable INVENIO_USER_EMAIL before runnning this script."
     echo "[ERROR] Example: export INVENIO_USER_EMAIL=info@inveniosoftware.org"
     exit 1
 fi
-if [ -v ${INVENIO_USER_PASS} ]; then
+if [[ -v INVENIO_USER_PASS ]]; then
     echo "[ERROR] Please set environment variable INVENIO_USER_PASS before runnning this script."
     echo "[ERROR] Example: export INVENIO_USER_PASS=uspass123"
     exit 1
 fi
-if [ -v ${INVENIO_POSTGRESQL_HOST} ]; then
+if [[ -v INVENIO_POSTGRESQL_HOST ]]; then
     echo "[ERROR] Please set environment variable INVENIO_POSTGRESQL_HOST before runnning this script."
     echo "[ERROR] Example: export INVENIO_POSTGRESQL_HOST=192.168.50.11"
     exit 1
 fi
-if [ -v ${INVENIO_POSTGRESQL_DBNAME} ]; then
+if [[ -v INVENIO_POSTGRESQL_DBNAME ]]; then
     echo "[ERROR] Please set environment variable INVENIO_POSTGRESQL_DBNAME before runnning this script."
     echo "[ERROR] Example: INVENIO_POSTGRESQL_DBNAME=invenio3"
     exit 1
 fi
-if [ -v ${INVENIO_POSTGRESQL_DBUSER} ]; then
+if [[ -v INVENIO_POSTGRESQL_DBUSER ]]; then
     echo "[ERROR] Please set environment variable INVENIO_POSTGRESQL_DBUSER before runnning this script."
     echo "[ERROR] Example: INVENIO_POSTGRESQL_DBUSER=invenio3"
     exit 1
 fi
-if [ -v ${INVENIO_POSTGRESQL_DBPASS} ]; then
+if [[ -v INVENIO_POSTGRESQL_DBPASS ]]; then
     echo "[ERROR] Please set environment variable INVENIO_POSTGRESQL_DBPASS before runnning this script."
     echo "[ERROR] Example: INVENIO_POSTGRESQL_DBPASS=dbpass123"
     exit 1
 fi
-if [ -v ${INVENIO_REDIS_HOST} ]; then
+if [[ -v INVENIO_REDIS_HOST ]]; then
     echo "[ERROR] Please set environment variable INVENIO_REDIS_HOST before runnning this script."
     echo "[ERROR] Example: export INVENIO_REDIS_HOST=192.168.50.12"
     exit 1
 fi
-if [ -v ${INVENIO_ELASTICSEARCH_HOST} ]; then
+if [[ -v INVENIO_ELASTICSEARCH_HOST ]]; then
     echo "[ERROR] Please set environment variable INVENIO_ELASTICSEARCH_HOST before runnning this script."
     echo "[ERROR] Example: export INVENIO_ELASTICSEARCH_HOST=192.168.50.13"
     exit 1
 fi
-if [ -v ${INVENIO_RABBITMQ_HOST} ]; then
+if [[ -v INVENIO_RABBITMQ_HOST ]]; then
     echo "[ERROR] Please set environment variable INVENIO_RABBITMQ_HOST before runnning this script."
     echo "[ERROR] Example: export INVENIO_RABBITMQ_HOST=192.168.50.14"
     exit 1
 fi
-if [ -v ${INVENIO_WORKER_HOST} ]; then
+if [[ -v INVENIO_WORKER_HOST ]]; then
     echo "[ERROR] Please set environment variable INVENIO_WORKER_HOST before runnning this script."
     echo "[ERROR] Example: export INVENIO_WORKER_HOST=192.168.50.15"
     exit 1
 fi
 
 # load virtualenvrapper:
-source $(which virtualenvwrapper.sh)
+source virtualenvwrapper.sh
```

Comment:
I'm having issues trying to run the virtualbox setup on fc24 (it requires an older kernel, that breaks everything else, and the images used don't support libvirt). Will try to figure out a way to test it...